src/scripts/ui/BuildingStorageComponent.tsx

```diff
@@ -114,6 +118,31 @@ export function BuildingStorageComponent({ gameState, xy }: IBuildingComponentPr
                </details>
             </li>
          </ul>
+         <div className="sep5"></div>
```

Comment:
The implementation here is problematic: because not all buildings can change capacity, and some buildings that can set capacity might not have `Storage Component` (components are reused). So this button should be at least next to the Capacity slider to make sure that only buildings can change capacity can use this button. Otherwise a player might get soft locked.
![image](https://github.com/user-attachments/assets/76eb3546-7a7e-461b-b944-64821cda4dbb)

Reply:
I did put it there to start with and then realised that those options are part of `ApplyToAllComponent` which is reused throughout the building, so putting it there was not an option.

I can put it above those options so that it sits above the Apply to all cog/gear.

Reply:
I feel a button takes too much vertical space? You can leave it as a button as I can adjust the styling later.
Can you also resolve the conflict so that I can merge this PR?

Reply:
I did rebase on top of main, so I believe that all conflicts should be resolved now.